crates/vm/src/arch/execution_mode/mod.rs

```diff
@@ -1,15 +1,15 @@
 use crate::{arch::VmExecState, system::memory::online::GuestMemory};
 
-pub mod e1;
 pub mod metered;
-pub mod tracegen;
+pub mod normal;
```

Comment:
I had to be referring to it as `pure` -- I'm also ok with normal but thought `pure` might be a bit less ambiguous?